mobius-core/src/main/java/com/spotify/mobius/Connectable.java

```diff
@@ -29,6 +30,10 @@
  * Mobius loop. In that case, the incoming values will be effects, and the outgoing values will be
  * events that should be sent back to the loop.
  *
+ * <p>Alternatively used in {@link Controller#connect(Connectable)} to connect a view to the
+ * controller. In that case, the incoming values will be events, and the outgoing values will be
```

Comment:
actually in this case the input `I` is the Model and the output `O` is the Event

Reply:
@anawara thanks missed that - updated